source/features/fit-textareas.css

```diff
@@ -2,3 +2,7 @@
 	max-height: none !important;
 	field-sizing: content;
 }
+
+textarea.rgh-fit-textareas.form-control {
```

Comment:
Don't all textareas have this class?

Reply:
the focused `textarea`s have this class:

```
js-comment-field js-paste-markdown js-task-list-field js-quick-submit FormControl-textarea CommentBox-input rgh-fit-textareas js-saved-reply-shortcut-comment-field input-monospace rgh-seen--7661699013
```

even if `height: auto` (without `!important`) was applied in future to the focused `textarea` elements, it would still not cause a problem for the manual resizing, which sets an inline `height`, which is higher specificity

Reply:
Selector not strong enough for me, it's 200px:

<img width="338" alt="Screenshot 3" src="https://github.com/refined-github/refined-github/assets/1402241/265c3df2-7d16-4d1d-bd12-c2b6ce19316d">

```suggestion
:root textarea.rgh-fit-textareas.form-control {
```

Should work

Reply:
ok sounds good. interesting, wonder why that didn't show up for me - maybe I didn't test correctly

Reply:
> wonder why that didn't show up for me

My guess is that the browser isn't consistent in the injection order of the styles.